docs/rules/object-curly-spacing.md

```diff
@@ -22,28 +22,28 @@ export { foo };
 
 ## Rule Details
 
-This rule aims to maintain consistency around the spacing inside of object literals. It also
-applies to EcmaScript 6 destructured assignment and import/export specifiers.
-
-It either requires or disallows spaces between those braces and the values inside of them.
-Braces that are separated from the adjacent value by a new line are exempt from this rule.
+This rule enforce consistent spacing inside braces of object literals, destructuring assignments, and import/export specifiers.
 
 ## Options
 
-There are two main options for the rule:
+This rule has a string option:
```

Comment:
Hmm, this style is a bit confusing to me. The multiple "this rule has an X option" makes it difficult for me to understand if there's just one option that can be a string or object, or if there are multiple options.

I'd much rather we start with "This rule has two options," as I think that's a lot clearer.